### ISSUE TITLE
Instruct users to downgrade to `v0.1.3.9014` if `renv.lock` is present

### ISSUE THREAD

**Issue by lorenzwalthert** [lorenzwalthert/precommit#346]
@smingerson for the short-term until #342 is resolved, I suggest to display a warning message in `autoupdate()` and friends that using {renv} is incompatible with `precommit > 0.1.3.9014`

**Comment by lorenzwalthert** [comment]
@smingerson can you 
```r
remotes::install_github("lorenzwalthert/precommit#344")
# restart R
precommit::autoupdate()
# try to commit
```
?

This temporarily downgrades the hook versions to not use `language: r`.

**Comment by lorenzwalthert** [comment]
Closed with `v0.2.2`.
